test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
+
+
+  Scenario: routes can define reverse routes with 'as'
+    Given a file "routes.coffee" with the contents
+      """
+      module.exports = ({ GET }) ->
+        GET '/some/route', to: 'test#index', as: 'foobar'
+      """
+    And an exprestive app using defaults
+    Then the function res.locals.paths.foobar() should return "/some/route"
+
+
+  Scenario Outline: restful routes define reverse routes automatically
+    Given a file "routes.coffee" with the contents
+      """
+      module.exports = ({ resources, GET }) ->
+        resources 'users'
+      """
+    And an exprestive app using defaults
+    Then the function res.locals.<LOCAL> should return "<VALUE>"
```

Comment:
What do you think about
```cucumber
Then I have a routing helper "res.locals.<LOCAL>" that returns "<VALUE>"
```

I think this table makes it more clear how it works:
```cucumber
Examples:
  | LOCAL            | VALUE      |
  | paths.users(123) | /users/123 |
```
than
```cucumber
Examples:
  | LOCAL       | ARGUMENT | VALUE      |
  | paths.users | 123      | /users/123 |
```